.github/workflows/lint-bpf-checks.yaml

```diff
@@ -133,7 +133,8 @@ jobs:
           fetch-depth: 0
       - name: Run BPF tests with code coverage reporting
```

Comment:
This title isn't accurate anymore :sweat_smile: Not sure if it's worth fixing.

Reply:
🤦 I've fixed and re-pushed.